client/my-sites/stats/overview.jsx

```diff
@@ -79,7 +79,9 @@ class StatsOverview extends Component {
 				.format( 'YYYY-MM-DD' );
 
 			if ( 0 === index || sitesSorted[ index - 1 ].periodEnd !== site.periodEnd ) {
-				overview.push( <DatePicker period={ period } date={ date } /> );
+				overview.push(
+					<DatePicker period={ period } date={ date } key={ `datepicker-${ index }` } />
```

Comment:
[Indexes are not recommended as key values](https://reactjs.org/docs/lists-and-keys.html#keys). If I understand this correctly there's a possibility that we display several `<DatePicker />` components. What about replacing `${ index }` with `${ side.ID }` ?

Reply:
This was another reason I prefer the `Fragment` 🙂

Reply:
Thanks for chiming in, folks. 

FWIW, this component already uses indexes for keys, which is why I went with this approach. While I'd love to obliterate the entire component and refactor it from scratch, this is not the purpose of this PR.

I'm happy to close it if y'all prefer a full refactor.